### DISCUSSION TITLE
No attribute class found for 'JetBrains\PhpStorm\Pure'

### DISCUSSION THREAD

**Discussion by handy-oss** [w7corp/easywechat#2595] [General]
在hyperf框架下使用会出现这个错误，加入ignore_annotations配置也一样
PHP Fatal error:  Uncaught Hyperf\Di\Exception\NotFoundException: No attribute class found for 'JetBrains\PhpStorm\Pure' in EasyWeChat\MiniApp\Application->getUtils() method in /data/vendor/hyperf/di/src/Annotation/AnnotationReader.php:266

**Comment by jbRotk** [comment]
php-swagger问题
